apollo-audit/apollo-audit-impl/src/main/java/com/ctrip/framework/apollo/audit/constants/ApolloAuditContextConstants.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.audit.constants;
+
+public interface ApolloAuditContextConstants {
+
+  String TRACER = "Apollo-Audit-Tracer";
```

Comment:
Delete this class,

and move all constant to ApolloAuditHttpHeaderConstants?

Reply:
what about move to a new class ApolloAuditConstants contains all constants?